src/constants.js

```diff
@@ -49,7 +49,8 @@ export const errorTypes = {
   ITEM_CHECKED_OUT: 4,
 };
 
-export const MAX_RECORDS = 40;
+export const MAX_RECORDS_FOR_CHUNK = 40;
+export const MAX_RECORDS = '10000';
```

Comment:
Is this conversion from number to string intentional?

Reply:
Huh, it seems weird that this clearly numeric value would every be specified as a string, but you are right about how it appears in other repos. I was mostly commenting on this repository, however: it was previously an int, now it's a string, and your PR doesn't mention this. 

I guess my point is less, "Does this work?" (clearly, it does) than "when you change things, please explain your reasoning."